controllers/kustomization_controller_test.go

```diff
@@ -55,6 +58,37 @@ var _ = Describe("KustomizationReconciler", func() {
 			err = k8sClient.Create(context.Background(), namespace)
 			Expect(err).NotTo(HaveOccurred(), "failed to create test namespace")
 
+			c := clientcmdapi.NewConfig()
```

Comment:
Converting from a rest.Config to a api.Config was actually a bit tricky and there does not seem to be an easy way to do this. We need to convert each of the parameters from one config to another.

Reply:
You may want to take a look at helm-controller (`internal` + the bits wiring up Helm), as there are some conversion utils there.